src/components/crossDevice/MobileConnected/index.js

```diff
@@ -7,22 +7,20 @@ import {preventDefaultOnClick} from '../../utils'
 
 const MobileConnected = ({message, submessage, back}) => {
   return (
-    <div>
+    <div className={theme.thickWrapper}>
```

Comment:
This screen looks like this on Chrome on my machine.
![screen shot 2017-11-22 at 11 59 20](https://user-images.githubusercontent.com/7127427/33126577-e7ccc172-cf7c-11e7-88a0-64bf33d8cad1.png)